src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -147,6 +148,9 @@ public Command parseCommand(String userInput) throws ParseException {
             case RedoCommand.COMMAND_WORD:
                 return new RedoCommand();
 
+            case ThemeCommand.COMMAND_WORD:
+                return new ThemeCommandParser().parse(arguments);
```

Comment:
Noticed that there was no handling for when the user treats the theme command as a complex command (commands that include person type tags `-pa` or `-sp`. Maybe you can add a useful error message instead of `Unknown command` as seen below

![Untitled](https://github.com/AY2324S1-CS2103T-W13-1/tp/assets/95404279/399fce7d-7a7c-49d3-969e-4e1bbdffc629)

Reply:
Yes, agreed. Seems like there is already an open issue related to this. When that is resolved, it should take care of this problem for all 'simple commands' simultaneously.